e2e-tests/authd-msentraid/common/broker.resource

```diff
@@ -0,0 +1,298 @@
+*** Settings ***
```

Comment:
What's the purpose of the `common` directory in `authd-msentraid`? AFAICT nothing else uses the files in that directory. Can we remove that directory and move its contents to `authd-msentraid/`?

Reply:
I reorganized the assets in the latest commit. What do you think?